apache/superset
# feat: Allow mutli-select values to be edited

### SUMMARY
When you mistype a filter value, there is no way to fix your mistake other than removing and re-adding it. This PR allows you to edit select component values by double-clicking on the filter tag.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/10563996/183101208-8d4e29b3-e498-4b9b-bb56-ff8a1fde1f3d.mp4


### TESTING INSTRUCTIONS
Add a filter value to a dashboard with the editable option enabled. Double click on the tag; an input should show up to allow you to modify the value. Hitting enter or clicking away from the input should save the value.

### ADDITIONAL INFORMATION
